osu.Game/Screens/Edit/Compose/Components/SelectionBox.cs

```diff
@@ -173,9 +173,6 @@ public string Text
         [BackgroundDependencyLoader]
         private void load()
         {
-            if (rotationHandler != null)
-                canRotate.BindTo(rotationHandler.CanRotate);
```

Comment:
This cannot be done.

Reply:
oh, i see now the problem. gonna be back with some changes.